keyboards/bemeier/bmek/bmek.c

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2020 bemeier
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "bmek.h"
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case BM_RST:  // RESET KEYBOARD
+            if (record->event.pressed) break;
+            for (int i = 0; i < RGBLED_NUM; i++) {
+                rgblight_sethsv_at(255, 255, 255, i);
+            }
```

Comment:
Why not just do this: 
```suggestion
            rgblight_setrgb(255, 0, 0);
```

Also, you could just use `RESET` rather than a custom keycode?

Reply:
Thanks for pointing this out, I wasn't aware of shutdown_user, took your suggestions and removed the custom keycode as well as the for loop here: 76a8869